hub/src/server/authentication.js

```diff
@@ -259,7 +283,15 @@ export function parseAuthHeader(authHeader: string) {
 }
 
 export function validateAuthorizationHeader(authHeader: string, serverName: string,
-                                            address: string) : string {
+                                            address: string, requireCorrectHubUrl?: boolean = false,
+                                            validHubUrls?: ?Array<string> = null) : string {
+  const serverNameHubUrl = `https://${serverName}`
```

Comment:
I think I know the answer, but just want to make sure---there's never going to be a world where we support `http`, right?  I don't think it would work in Web browsers anyway, unless the app itself was served via `http`.

Reply:
Yep -- not a world that Gaia wants to live in.